### DISCUSSION TITLE
Dropdown in Navigation

### DISCUSSION THREAD

**Discussion by taker99** [requarks/wiki#6115] [Help / Questions] [answered]
Hello,

I would like to add dropdowns to my navigation, how do I do that?

Greetings

```
OS: Debian 11 and Docker
Wiki.js version: 2
```

**Comment by NGPixel** [comment, marked as answer]
Not supported yet.

**Reply by 2b0x** [reply]
Hi, i want to add a button in the toolbar. Could you please grant customization permissions to the toolbar in Word document formatting, or provide the corresponding source code?
